ITISFoundation/osparc-simcore
# 📝 Doc conventions, 🔨 issue templates and ♻️ error messages

## What do these changes do?

- 📝 doc conventions
- 🔨 template issues and tools for releases (originally in ITISFoundation/osparc-issues). This is a prerequisite before all the release issues are moved to this repo
    -  Converts markdown to forms https://github.com/pcrespov/sandbox-python/issues/new/choose
- ♻️ Following up pair-debugging with @odeimaiz : makes invitation error more human-readable


## Related issue/s

-  #3943: adds pre-release issue template to this repo

## How to test

 - check to create in issue using  https://github.com/pcrespov/sandbox-python/issues/new/choose
